core/model/modx/processors/element/propertyset/updatefromelement.class.php

```diff
@@ -9,44 +10,39 @@
 
 class modPropertySetUpdateFromElementProcessor extends modPropertySetUpdateProcessor {
     public $languageTopics = array('propertyset', 'category', 'element');
-
-    /** @var modPropertySet|null */
-    public $object;
```

Comment:
> Why you removed this? It serves for autocompletion in IDE
Because if $this->classKey == 'modSnippet'; we need not get elements.

You may correct this code, if will work OK.